modules/apps/object/object-dynamic-data-mapping-storage/src/main/java/com/liferay/object/dynamic/data/mapping/storage/internal/ObjectDDMStorageAdapter.java

```diff
@@ -143,16 +146,25 @@ public DDMStorageAdapterSaveResponse save(
 			DDMStorageAdapterSaveResponse ddmStorageAdapterSaveResponse =
 				_ddmStorageAdapter.save(ddmStorageAdapterSaveRequest);
 
+			DDMFormInstance ddmFormInstance =
+				ddmStorageAdapterSaveRequest.getDDMFormInstance();
+
+			DDMFormInstanceSettings ddmFormInstanceSettings =
+				ddmFormInstance.getSettingsModel();
+
+			long objectDefinitionId = Long.parseLong(
+				ddmFormInstanceSettings.objectId());
```

Comment:
@gabrielwas, considering that Objects API uses `objectDefinitionId` instead of `objectId`, shouldn't we use this same name on form settings to keep consistency?

Reply:
You're right! Went unnoticed to me :D thanks!